packages/wdio-protocols/protocols/appium.json

```diff
@@ -1308,6 +1308,41 @@
         "description": "The type of action to perform (e.g. moveTo, release, press, tap, wait)",
         "required": true
       }],
+      "examples": [
+        [
+          "// do a horizontal swipe by percentage",
+          "const { width, height } = driver.getWindowSize();",
```

Comment:
achorPercentage and endPercentage do not exist in the example

Reply:
added missing vars